Sources/Client/Client/Client+Setup.swift

```diff
@@ -179,9 +179,16 @@ extension Client {
         let completion = completion == nil ? nil : { result in DispatchQueue.main.async { completion?(result) } }
         
         do {
-            let webSocket = try setupWebSocket(user: user, token: token)
+            let webSocketProvider = try createWebSocketProvider(user: user, token: token)
+            let webSocketOptions = stayConnectedInBackground ? WebSocketOptions.stayConnectedInBackground : []
+            
+            if webSocket.provider.request.url == .placeholder {
+                webSocket = createWebSocket(provider: webSocketProvider, options: webSocketOptions)
+            } else {
+                webSocket.provider = webSocketProvider
+            }
```

Comment:
So we only create the websocket for the first time, and in consequent times (logout/login) we use the old websocket